From bc8fab9521dbd5219e65704e8cc3320f51d442f9 Mon Sep 17 00:00:00 2001
From: Yunwei Zhang <yunwei.zhang@intel.com>
Date: Fri, 29 Jan 2016 10:36:30 -0800
Subject: [PATCH] Revert "FOR_UPSTREAM [VPG]: drm/i915: count engine resets"

Patch is causing 3 OpenGL conformance test cases to fail
robustness_es2.advanced.resets.threeContexts,
robustness_es2.advanced.resets.drawAfterReset3,
robustness_es2.advanced.resets.drawAfterReset2
This reverts commit 4c926072a6e729d2ec54a566177fe22ee483f283.

Change-Id: I4aa663c24a8ea33d9b8a94e58d8378883314b7c5
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-18177
Signed-off-by: Yunwei Zhang <yunwei.zhang@intel.com>
Reviewed-on: https://android.intel.com:443/464285
---
 drivers/gpu/drm/i915/intel_uncore.c | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_uncore.c b/drivers/gpu/drm/i915/intel_uncore.c
index 6e666ad..7d96124 100644
--- a/drivers/gpu/drm/i915/intel_uncore.c
+++ b/drivers/gpu/drm/i915/intel_uncore.c
@@ -1084,15 +1084,10 @@ int i915_get_reset_stats_ioctl(struct drm_device *dev,
 	}
 	hs = &ctx->hang_stats;
 
-	if (capable(CAP_SYS_ADMIN)) {
-		int i;
-		struct intel_engine_cs *ring;
+	if (capable(CAP_SYS_ADMIN))
 		args->reset_count = i915_reset_count(&dev_priv->gpu_error);
-		for_each_ring(ring, dev_priv, i)
-			args->reset_count += ring->hangcheck.total;
-	} else {
+	else
 		args->reset_count = 0;
-	}
 
 	args->batch_active = hs->batch_active;
 	args->batch_pending = hs->batch_pending;
-- 
1.9.1

